doc/examples/animations-pulsating-sphere.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Animations of a Pulsating Sphere"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sfs\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import animations_pulsating_sphere as ani\n",
+    "from IPython.display import display, Image, HTML"
```

Comment:
No need to import `display`. That's imported implicitly by recent versions of IPython.

Reply:
OK, thanks.